lib/less.js

```diff
@@ -18,7 +18,7 @@ var fs = require('fs');
 var url = require('url');
 var path = require('path');
 var async = require('async');
-var assign = require('object-assign');
```

Comment:
As `object-assign` is not used anymore, please also remove it from the dependencies.